.github/workflows/chromatic.yml

```diff
@@ -48,4 +48,9 @@ jobs:
     # (else) Run this step on dependabot PRs.
     - name: Skip Chromatic builds
       if: ${{ github.actor == 'dependabot[bot]' }}
-      run: yarn chromatic --skip
+      uses: chromaui/action@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        projectToken: ${{ secrets.CHROMATIC_APP_CODE }}
+        exitZeroOnChanges: true
+        skip: true
```

Comment:
Should this still be there? What is it skipping? Can you perhaps add a comment so that it's clear why we need this?

Reply:
Fair enough! It's added so the Chromatic build reports back to GH, but that we don't need to process any stories inside this PR. That saves running some unnecessary commands (that means in saving time when running this workflow on these PRs).